tests/test_urldispatch.py

```diff
@@ -486,7 +486,7 @@ def test_add_static_quoting(router: web.UrlDispatcher) -> None:
     )
     assert router["static"] is resource
     url = resource.url_for(filename="/1 2/файл%2F.txt")
-    assert url.path == "/пре %2Fфикс/1 2/файл%2F.txt"
+    assert url.path == "/пре /фикс/1 2/файл%2F.txt"
```

Comment:
I'm not clear why we want to change this (and the introduction of _requote_path())?

Feels to me like this might be an incorrect URL, given the add_statci() appears to be looking for a literaly `%2F`.

Reply:
It changes because I removed the ignore of `/` for `.path` in https://github.com/aio-libs/yarl/pull/1150/files#diff-657d7257a2bac78ab69f50e60aa9cef5620adc8cd7f902d4e4cbeb0c6e78a434R228

Do we want to keep the ignore for `.path` and `.path_safe` ?

Reply:
No, we should revert it. I'm just thinking about it wrong. The important thing is the raw URL that is given to the user, which is tested on the next line. The .path output here isn't so important.

Reply:
Ok I will split that PR for yarl so its more obvious its being reverted